htsohm/material_files.py

```diff
@@ -72,64 +72,46 @@ def write_seed_definition_files(run_id, number_of_atomtypes):
     sigma_limits            = config["sigma_limits"]
     max_charge              = config["charge_limit"]
     elem_charge             = config["elemental_charge"]
-    raspa2_dir              = config["raspa2_dir"]
-    ff_dir   = os.path.join(raspa2_dir, 'share', 'raspa', 'forcefield')         # forcefield-files
-    mat_dir  = os.path.join(raspa2_dir, 'share', 'raspa', 'structures', 'cif')  # .cif-files
 
-    ########################################################################
-    material = Material(run_id)
-    material_name = run_id + '-' + material.uuid
-    material.generation = 0
-
-    def_dir = os.path.join(ff_dir, material_name)       # directory for material's force field
-    os.mkdir(def_dir)
-    force_field_file = os.path.join(def_dir, 'force_field.def')      # for overwriting LJ-params
-    write_force_field(force_field_file)
+    htsohm_dir = os.path.dirname(os.path.dirname(htsohm.__file__))
+    run_dir = os.path.join(htsohm_dir, run_id)
+    material_dir = os.path.join(run_dir, 'pseudo_materials')
+    if not os.path.exists(material_dir):
+        os.mkdir(material_dir)
 
     ########################################################################
-    # define pseudo atom types by randomly-generating sigma and epsilon values
-    #
-    # NOTE :  assignment of partial charges is not supported in this version,
-    #         and not necessary when modelling gases without dipole moments
-    #         (such as methane). Charge assignment and mutation will appear
-    #         in the next stable release.
-    #
-    atom_types = []
+    db_row = Material(run_id)
+    db_row.generation = 0
+    material = PsuedoMaterial(db_row.uuid)
```

Comment:
instances of Material (database row) and PseudoMaterial (structural data object) classes are now consistently named `*material` and `*pseudo_material` respectively